Product.php

```diff
@@ -130,8 +130,8 @@ public function End() {
         $this->SetProductStatus(self::PSTATUS_ENDED);
     }
 
-/* @{ */
-
+/* @{

```

Comment:
Is there a reason for this change?

Reply:
My IDE give me an error for putting them in one line, and I have no clue why.